stdlib/public/stubs/SwiftNativeNSXXXBase.mm.gyb

```diff
@@ -110,6 +134,20 @@ SWIFT_RUNTIME_STDLIB_API
 @end
 % end
 
+// Array -> NSArray bridging was unfortunately inlinable in 5.0/5.1, so there
+// is a possibility that existing apps would not hit the call to
+// swift_stdlib_connectNSBaseClasses. To work around this, we also call it the
+// first time a bridged array is sent an objc message
```

Comment:
Does this actually only happen if you bridge?

Reply:
Nope, which is why it required additional special casing. We can't assert if this one fails. Other suggestions extremely welcome, this is what Karoy and I were able to come up with.

Reply:
(specifically, the metadata accessor calls -class. SIGH)

Reply:
We discussed this and Jordan pointed out that we can actually get away with not doing it :)